about.html

```diff
@@ -0,0 +1,151 @@
+                    <i class="fab fa-bitcoin">Bitcoin</i>
+                </a>
+            </div>
+            <ul class="top-nav">
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">
+                        <i class="fab fa-twitter"></i>
+                    </a>
+                </li>
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">
+                        <i class="fab fa-facebook-f"></i>
+                    </a>
+                </li>
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">English</a>
+                </li>
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">Logout</a>
+                </li>
+            </ul>
+            <ul class="nav-list">
+                <li class="nav-item">
+                    <div class="icon">
+                       <i class="fas fa-times" id="close-icon"></i>
+                    </div>
+                </li>
+              <li class="nav-item"><a href="index.html" class="nav-link">Home</a></li>
+              <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+            </ul>
+        </nav>
+        <div class="hero-container">
+            <div class="hero-about-content">
+                <h4 class="hero-about-content__heading">
+                   BLOCK CHAIN SUMMIT 2021
+                </h4>
+                <p class="hero-about-content__desc bg-white">
+                 The Block Chain Summit brings together community of
+                 experts, academics, and activists who comprise the Creative
+                 Commons affliate network in a different country
+                 every two years. This year, we hope to expand our invitation list
+                 including organizations and individuals
+                </p>
+                <div class="hero-about-content-contact">
+                    <p>
+                       Please contact us per Email for any
+                       further questions about Block Chain Summit
+                       2021!
+                    </p>
+                    <a href="#">blockchainsummit@gmail.com</a>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main>
+        <section>
+            <div class="container">
+                <div class="summits">
+                    <h2 class="summits-heading">
+                        See the past Block Chain Summits
+                    </h2>
+                    <p class="summits-text">
+                        Take a look at the last two Block Chain
+                        Summit which took place in China and in Dubai
+                    </p>
+                    <ul class="summits-list">
+                        <li class="summits-list-item">
+                            <img src="./images/conference_one.jpg" alt="">
```

Comment:
- [ ] The previous reviewer also mentioned the alt attribute but it has not been addressed. The alt tag describes the image or at least states what the image is trying to convey to the person seeing the page. In addition, users of screen magnification may not be able to get a complete view of the image and thus need the alt tag to let them know what the purpose of the image is. The same goes for `line 98`